conda-forge.yml

```diff
@@ -1,4 +1,4 @@
-bot: {automerge: true}
+bot: {automerge: false}
```

Comment:
Thanks @dopplershift for the dependencies update commit. I removed automerge to avoid future bad merges before we do that. Ideally we should be using grayskull to update stuff and use automerge in a safer way.

Reply:
Makes sense. If we can get this to pass (*Cough* Travis), that should allow the 3.10 migration to proceed a bit more.